bin/pycbc_multi_inspiral

```diff
@@ -648,8 +671,8 @@ with ctx:
                 continue
 
             # Loop over (short) time-slides, staring with the zero-lag
-            for slide in range(args.num_slides + 1):
-                logging.info("Analyzing slide %d/%d", slide, args.num_slides)
+            for slide in range(num_slides + 1):
```

Comment:
We have the + 1 for the zero-lag in the `num_slides` formula so the +1 here should be removed.  Related to this, see my comment in `def slide_limiter(args):` and for `slide_ids`.